src/normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Using normalize.css is usually a good solution for large projects, but for this small task it's redundant. Most of tags mentioned in this file don't even exist in your HRML. And we only need to reset margin for `body`.